docs/src/getting-started/quickstart.md

```diff
@@ -6,91 +7,141 @@ In this tutorial you will:
 2. Create, build, and deploy an indexer to an indexer service hooked up to Fuel's `beta-3` testnet.
 3. Query your newly created index for data using GraphQL.
 
-## 1. Setting up your environment
+## 1. Prerequisites
 
-In this Quickstart, we'll use Fuel's toolchain manager [`fuelup`](https://github.com/FuelLabs/fuelup) in order to install the `forc-index` component that we'll use to develop our indexer.
+In this Quickstart, we'll use Fuel's toolchain manager [`fuelup`](https://github.com/FuelLabs/fuelup) in order to install the `forc-index` component that we'll use to develop our indexer. There are a lot of prerequisites so please read through each installation carefully.  
 
-### 1.1 Install `fuelup`
+1. [Install the Rust toolchain](https://www.rust-lang.org/tools/install).
 
-To install fuelup with the default features/options, use the following command to download the fuelup installation script and run it interactively.
+2. [Install PostgreSQL](https://www.postgresql.org/docs/).
 
-```bash
-curl \
-  --proto '=https' \
-  --tlsv1.2 -sSf https://fuellabs.github.io/fuelup/fuelup-init.sh | sh
+Note: macOS you can install the latest version of PostgreSQL through `brew` by simply running:
+
+<!-- markdownlint-disable MD014 -->
+```console
+$ brew install postgresql
 ```
+<!-- markdownlint-restore -->
 
-> If you require a non-default `fuelup` installation, please [read the `fuelup` installation docs.](https://github.com/FuelLabs/fuelup)
+3. [Install the Fuel toolchain](https://github.com/FuelLabs/fuelup).
 
-### 1.2 WebAssembly (WASM) Setup
+Make sure you have the latest version of `fuelup` by running the following command:
 
-Indexers are typically compiled to WASM so you'll need to have the proper WASM compilation target available on your system. You can install this target using `rustup`:
+Note: older versions of `fuel-indexer` might not work in this guide
```

Comment:
Consider moving this line under the `fuelup self update` line and changing to:
> Check that you have the latest available version of `fuel-indexer` by ensuring that your toolchain is up-to-date.

It's a bit more helpful and doesn't make an assumption of the toolchain that they're using.